api/datasets.go

```diff
@@ -20,17 +20,23 @@ func (c *CodeListAPI) getCodeDatasets(w http.ResponseWriter, r *http.Request) {
 
 	datasets, err := c.store.GetCodeDatasets(ctx, codeListID, edition, code)
 	if err != nil {
+		logData["err"] = err
+		log.InfoCtx(ctx, "failed to get datasets list", logData)
```

Comment:
shouldn't this be log.ErrorCtx?

Reply:
ErrorCtx doesn't feel as clear, because I can only pass err and data - so my ability to pass the string message is removed, and i lose context of where in the application the failure happened.